api/src/services/applications/application_validation.py

```diff
@@ -164,21 +148,61 @@ def _get_json_rule_config_for_action(action: ApplicationAction) -> JsonRuleConfi
 
 def validate_application_form(
     application_form: ApplicationForm, action: ApplicationAction
-) -> list[ValidationErrorDetail]:
-    """Validate an application form, and set the current application form status"""
+) -> tuple[list[ValidationErrorDetail], list[ValidationErrorDetail]]:
+    """Validate an application form, and set the current application form status
+
+    Returns:
+        tuple: (form_validation_errors, inclusion_errors)
+            - form_validation_errors: errors that should be wrapped in APPLICATION_FORM_VALIDATION
+            - inclusion_errors: MISSING_INCLUDED_IN_SUBMISSION errors that should be returned directly
+    """
     form_validation_errors: list[ValidationErrorDetail] = []
+    inclusion_errors: list[ValidationErrorDetail] = []
 
     context = JsonRuleContext(application_form, config=_get_json_rule_config_for_action(action))
     process_rule_schema_for_context(context)
     form_validation_errors.extend(context.validation_issues)
 
     # TODO pre/post-populate should update the value here
 
-    form_validation_errors.extend(
-        validate_json_schema_for_form(application_form.application_response, application_form.form)
-    )
+    # Check if the form is required
+    is_required = is_form_required(application_form)
+
+    # Handle validation based on form requirement and is_included_in_submission flag
+    should_run_json_schema_validation = True
+
+    if not is_required:
+        # For non-required forms, check is_included_in_submission
+        if application_form.is_included_in_submission is None:
+            # If the form has no content and is_included_in_submission is None,
+            # treat it as not included in submission (don't validate)
+            if len(application_form.application_response) == 0:
+                should_run_json_schema_validation = False
```

Comment:
I think we shouldn't have this `len(application_form.application_response) == 0` check. If a form isn't required, but hasn't set the `is_included_in_submission` flag, its just an error regardless of whether they've started it.